File: unity_game/Assets/Scripts/player_collider.js
#pragma strict
var seven_11:boolean;
var postOffice:boolean;
var mcdonald: boolean;

function Start () 
{
	seven_11 = false;
	postOffice = false;
}

function Update () 
{

}

function OnTriggerEnter(collision : Collider)
{
	if(collision.name=="7-11")
	{
		seven_11 = true;
		Debug.Log("7-11 true");
	}
	else if(collision.name=="postoffice")
	{
		postOffice = true;
		Debug.Log("post true");
	}
	else if(collision.name=="mcdonald")
	{
		mcdonald = true;
		Debug.Log("m true");
	}
	
}

function OnGUI()
{
	if(seven_11==false)
	GUILayout.Label(" 7-11 not found");
	else if(seven_11==true)
	GUILayout.Label(" 7-11 is found");
	
	if(postOffice==false)
	GUILayout.Label(" post office not found");
	else if(postOffice==true)
	GUILayout.Label(" post Office is found");
	
	if(mcdonald==false)
	GUILayout.Label(" mcdonald not found");
	else if(mcdonald==true)
	GUILayout.Label(" mcdonald is found");
}

File: unity_game/Assets/Scripts/PlayerCollider.cs
using UnityEngine;
using System.Collections;

public class PlayerCollider : MonoBehaviour
{
	
	public bool	seven_11;
	public bool	 postOffice;
	public bool	 mcdonald;
	
	public UICheckbox cb711;
	public UICheckbox cbMc;
	public UICheckbox cbPost;
	
	// Use this for initialization
	void Start ()
	{
		seven_11 = false;
		postOffice = false;
		mcdonald = false;
	}
	
	// Update is called once per frame
	void Update ()
	{
	
	}
	
	void	OnTriggerEnter (Collider collision)
	{
		if (collision.name == "7-11") {
			seven_11 = true;
			cb711.isChecked=true;
			Debug.Log ("7-11 true");
		} else if (collision.name == "postoffice") {
			postOffice = true;
			cbPost.isChecked=true;
			Debug.Log ("post true");
		} else if (collision.name == "mcdonald") {
			mcdonald = true;
			cbMc.isChecked=true;
			Debug.Log ("m true");
		}
	}
}
